radio/src/targets/taranis/CMakeLists.txt

```
@@ -507,6 +507,10 @@ else()
   message( FATAL_ERROR "Unknown CPU_TYPE_FULL" )
 endif()
 
+if(TRANSLATIONS STREQUAL UA AND NOT TARGET_FLASH_SIZE STREQUAL 1M)
```

```suggestion
if(NOT TARGET_FLASH_SIZE STREQUAL 1M AND (TRANSLATIONS STREQUAL RU OR TRANSLATIONS STREQUAL UA))
```

If you're going to do that, might as well do `RU` for completeness as it that suffers from the same issue. I tried building all languages for TX12MK2 (except for CN, JP, TW), and it was the only other one to fail.

I'll leave the rebase for you if you want to consider Phil's comment.